src/ray/gcs/asio_test.cc

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+
+#include "gtest/gtest.h"
+
+#include "asio.h"
+
+boost::asio::io_service io_service;
+
+class TestRedisAsioClient : public ::testing::Test {
+ public:
+  TestRedisAsioClient() {
+    int r = system("redis-server > /dev/null &");
```

Comment:
This will in general find a different redis-server from the one we installed, right?

Reply:
Yeah, this is only for the test, we don't use modules there.

Reply:
It may not run if the user hasn't done `sudo apt-get install redis` or something like that.

Reply:
I added a comment to the test